src/shared/scrapers/USA/WI/index.js

```diff
@@ -176,14 +176,18 @@ const scraper = {
       const $ = await fetch.headless(this.url);
       const $table = $('#covid-county-table').find('table');
       const $trs = $table.find('tbody > tr:not(:last-child)');
+
       $trs.each((index, tr) => {
         const $tr = $(tr);
+
         regions.push({
           county: geography.addCounty(parse.string($tr.find('td:first-child').text())),
-          cases: parse.number($tr.find('td:last-child').text())
+          cases: parse.number($tr.find('td:nth-child(2)').text()),
+          deaths: parse.number($tr.find('td:nth-child(3)').text())
         });
       });
       {
+        // needs CR? not seeing this table on 3/26
```

Comment:
I'm seeing 0 from this one

Reply:
`yarn timeseries 2020-3-22 -l "WI"` all zeroes for cases

Reply:
`yarn timeseries 2020-3-22 -l "WI, USA"` is showing only zeroes for 3/26 and 3/25 for me (probably when page changed)

Reply:
Can you fix 3/26 and 3/25 too?